Makefile

```diff
@@ -41,7 +46,20 @@ ifeq ($(NON_MATCHING),1)
   COMPARE := 0
 endif
 
+# Version-specific settings
+ifeq ($(VERSION),gc-eu-mq)
+  CFLAGS += -DNON_MATCHING -DNDEBUG
+  CPPFLAGS += -DNON_MATCHING -DNDEBUG
+  OPTFLAGS := -O2 -g3
+  COMPARE := 0
```

Comment:
The "build a non matching rom at first and work towards OK" approach is new (to us at least), and I don't think it's been properly discussed and agreed upon

Plus idk if gc-eu-mq is the version we want to do next. gc-eu-dbg-* are other candidates

Or I missed both discussions of the above two points.

In any case I think this PR can solely focus on moving `build/` to `build/gc-eu-mq-dbg/` (in a way that involves a version variable ofc)

Reply:
That's fair, I removed all mentions of gc-eu-mq.